### PR TITLE
New package: mkp224o-1.5.0

### PR DESCRIPTION
**Pull request by hamlolum**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

 
#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc

### CODE REVIEW

File: srcpkgs/mkp224o/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'mkp224o'
+pkgname=mkp224o
+version=1.5.0
+revision=1
+archs="x86_64"
+build_style=gnu-configure
+configure_args="--enable-amd64-51-30k"
```
Comment (Chocimier) [inline srcpkgs/mkp224o/template:7]: Remove this, will compile for all architectures.

File: srcpkgs/mkp224o/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+makedepends="libsodium libsodium-devel"
+short_desc="Vanity address generator for ed25519 onion services"
+maintainer="hamlolum <hamlolum@tilde.club>"
+license="Public Domain"
```
Comment (Chocimier) [inline srcpkgs/mkp224o/template:11]: CC0-1.0

File: srcpkgs/mkp224o/template [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'mkp224o'
+pkgname=mkp224o
+version=1.5.0
+revision=1
+archs="x86_64"
+build_style=gnu-configure
+configure_args="--enable-amd64-51-30k"
+makedepends="libsodium libsodium-devel"
```
Comment (Chocimier) [inline srcpkgs/mkp224o/template:8]: *-devel packages depend on non-devel, remove libsodium

**Comment by hamlolum** [comment]
I have made the requested changes.

**Pull request merged**